src/cli.rs

```diff
@@ -23,6 +24,11 @@ fn get_macaroon_path_default(network: &str) -> PathBuf {
         .join(format!(".lnd/data/chain/bitcoin/{network}/admin.macaroon"))
 }
 
+fn print_and_exit(msg: &str) -> ! {
```

Comment:
I think we should do better on this function.
1. [Maybe instead of printing, we should send the message into stderr](https://github.com/lndk-org/lndk/issues/185)
2. Then we can call something like "exit_gracefully" or something that requires an error message that is printed to the stderr